webapp/graphite/render/datalib.py

```diff
@@ -106,15 +110,80 @@ def getInfo(self):
       'pathExpression' : self.pathExpression,
     }
 
```

Comment:
General comment: this is adding a lot of code that should really be isolated in remote.py (for me, all this remote finder thing is just "one" of the finders that we can have).

Maybe it's a good time to start moving more code to remote.py ?